bedrock/foundation/redirects.py

```diff
@@ -2,6 +2,12 @@
 
 redirectpatterns = (
 
+    redirect(r'^foundation/$', 'https://foundation.mozilla.org/en/'),
```

Comment:
Please omit the `en` locale in the redirect destinations here and below. If we hard-code them like this, then we can only ever redirect people to the English version of `foundation.mozilla.org`.